labs/a8.md

```diff
@@ -0,0 +1,736 @@
+---
+title: Lab 8 - Security Fundamentals
+layout: lab
+---
+
+# Security
+
+In this note, we will briefly cover a variety of topics that are of interest to 
+those studying computer security. 
+
+There are many aspects to security, and the field spans a number of 
+disciplines. We will cover the following:
+
+* Security Building Blocks
+  * Encryption - Symmetric and Public-Key Crypto
+  * Certs, Signatures
+  * Hashing
+* File Security
+* Network Security
+
+## Introduction
+
+The most important thing to remember when designing secure systems is 
+understanding your threat model. No system is guaranteed to be secure or able 
+to withstand all attacks, nor is this even possible in the face of extreme 
+adversaries. However, you can (and should) take precautions against the threats 
+you are likely to face. Balancing the need for authorized users to get access 
+to the system while keeping unauthorized users out is very easy to get wrong. 
+Fortunately, smart people have distilled the principles of security down to a 
+few axioms, covered very well in the first [lecture note][cs161_1] of CS 161 
```

Comment:
I don't think anyone is going to read this since there's already tons of stuff to read in this lab. I'd make it "additional reading" instead.